### DISCUSSION TITLE
Execution failed for task ':stripe_android:compileDebugJavaWithJavac'.

### DISCUSSION THREAD

**Discussion by uzairleo** [flutter-stripe/flutter_stripe#1713] [Q&A]
**Describe the bug**
I am using flutter stripe in flutter project having version 3.16.7 
I do checked all the closed issue related to this error and change my grade to latest 8.1 and also updated my android SDK version to 17 But still I am facing this issue and stuck with it for more than 15 hrs 
<img width="761" alt="Screenshot 2024-04-03 at 1 02 57 PM" src="https://github.com/flutter-stripe/flutter_stripe/assets/52172428/0cf13afc-6cf8-4aef-9853-fdac6200452d">

<img width="497" alt="Screenshot 2024-04-03 at 12 59 57 PM" src="https://github.com/flutter-stripe/flutter_stripe/assets/52172428/c5d5fc4c-ba05-471d-93d1-0ca9d366104b">
<img width="610" alt="Screenshot 2024-04-03 at 1 00 39 PM" src="https://github.com/flutter-stripe/flutter_stripe/assets/52172428/4c276c5e-cdfc-4234-a7ed-293151c6cc29">
<img width="651" alt="Screenshot 2024-04-03 at 1 01 04 PM" src="https://github.com/flutter-stripe/flutter_stripe/assets/52172428/3e292e85-54e0-428b-93d4-16d8e450c34d">

ISSUE I am facing still 
<img width="594" alt="Screenshot 2024-04-03 at 1 01 21 PM" src="https://github.com/flutter-stripe/flutter_stripe/assets/52172428/ffe73d07-bdba-429d-b245-bbeb04c54c0c">

**Smartphone / tablet**
 - Device: Android
 - OS: [e.g. iOS 13, Android 10]
 - Package version: 10.1.1
 - Flutter version 3.16.7

**Additional context**

**Comment by uzairleo** [comment]
For more details here is my Gradle version as well 
<img width="518" alt="Screenshot 2024-04-03 at 2 20 25 PM" src="https://github.com/flutter-stripe/flutter_stripe/assets/52172428/2b1720f0-9950-41c0-b4b7-ef40a9a60765">

**Comment by DavidEcclesPH** [comment]
seeing the same after attempting to update to 10.1.1. Builds on iOS, but throws the same error as above on both (an old android 8) device and Pixel 5 API 31 simulator.

Execution failed for task ':stripe_android:compileDebugJavaWithJavac'.
> error: invalid source release: 17

**Comment by DavidEcclesPH** [comment]
@uzairleo Updating android studio to 2023.2.1 has fixed this for me 👍

**Comment by uzairleo** [comment]
Are u sure ? Its related to android studio version ???

**Comment by DavidEcclesPH** [comment]
@uzairleo definitely fixed 👍 I read here flutter used the version of Java that Android Studio uses, so related to Java but indirectly Android Studio! https://stackoverflow.com/questions/77624676/stripe-androidcompilereleasejavawithjavac-error-invalid-source-release-17

**Comment by Rushi-Gandhi** [comment]
I managed to address the issue with the "task :stripe_android:compileDebugJavaWithJavac FAILED" error by implementing a solution in my project. I included the following dependency override in the configuration:

```yaml
dependency_overrides:
  stripe_android: "9.5.0+1"
```

This modification proved effective for me. Feel free to test it in your setup and inform me if it resolves the issue on your end as well.
